tests/unit/Codeception/Module/Db/DbCleaningTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+abstract class DbCleaningTest extends \PHPUnit_Framework_TestCase
```

Comment:
I was just following the "best practice" for PHPUnit, is the trait as a collection of test something that you guys do? I'm not a PHPUnit guy neither but I found that the abstract class is how PHP folks like to do it. In Ruby I would do "it_behaves_like" using rspec shared examples.... https://relishapp.com/rspec/rspec-core/docs/example-groups/shared-examples